From 623bec18a0404a7e50458fa888b1582ae1f4089d Mon Sep 17 00:00:00 2001
From: Li Jun <b47624@freescale.com>
Date: Mon, 23 Jun 2014 15:39:35 +0800
Subject: [PATCH] ENGR00319720-1 usb: gadget: add host_request_flag in
 usb_gadget for OTG

This patch adds host_request_flag in usb_gadget to store host request
information from application.

Signed-off-by: Li Jun <b47624@freescale.com>
---
 include/linux/usb/gadget.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/gadget.h b/include/linux/usb/gadget.h
index c3a6185..3b17e99 100644
--- a/include/linux/usb/gadget.h
+++ b/include/linux/usb/gadget.h
@@ -562,6 +562,7 @@ struct usb_gadget {
 	unsigned			b_hnp_enable:1;
 	unsigned			a_hnp_support:1;
 	unsigned			a_alt_hnp_support:1;
+	unsigned			host_request_flag:1;
 	unsigned			quirk_ep_out_aligned_size:1;
 };
 #define work_to_gadget(w)	(container_of((w), struct usb_gadget, work))
-- 
1.8.0

